src/locales/en/translation.json

```diff
@@ -160,6 +160,7 @@
   "pager.of": "of",
   "pager.rows": "Items per page:",
   "personalization": "Personalization",
+  "personalization.autoDetectColorScheme": "Sync with Computer",
```

Comment:
```suggestion
```

Doesn't look like this is used anymore after moving to a dropdown instead of an additional toggle/switch